api.md

```diff
@@ -53,5 +50,70 @@
   "stayId": "v3456yv456v46",
   "status": "pending"
 }
+```
+
+### API to signup:
+
+`/signup` POST
+
+### Parameters
+
+| param |   description   | data type | examples |
+|------------|-----------|------------|-----------|
+| __username__ |  The username as input by the user | String | "brianG123" |
+| __password__  |  The user's desired password as input by the user   | String | "writer123" |
+| __email__  |  Date to start the stay in UTC format  |  String  | "brianG@random.edu" |

```

Comment:
take out username, or mention that there are plans to deprecate
and email desc seems off